src/data/queries/tests/__snapshots__/pressRelease.test.tsx.snap

```diff
@@ -41,7 +41,7 @@ exports[`node--press_release formatData output formatted data 1`] = `
   "fullText": "<p>We invite you to come and read our 2019 Annual Report. </p>",
   "id": "6153ed5b-85c2-4ead-9893-3d656ad5d758",
   "introText": "We invite you to come and read our 2019 Annual Report. ",
-  "lastUpdated": "2021-04-12T14:25:27+00:00",
+  "lastUpdated": "2021-04-12T14:27:39+00:00",
```

Comment:
Because `lastUpdated` was changed in `src/lib/drupal/query.ts`, this value changed as well.